.github/workflows/integration-tests.yml

```diff
@@ -311,6 +315,91 @@ jobs:
           else
             echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
           fi
+      - name: Check for "enable tracing" label
+        id: check-label
+        run: |
+          label=$(jq -r '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
```

Comment:
Thanks for the suggestion! I gave this a try, but it failed to work for CI runs. It may have worked for the merge queue though. I found an alternative solution.